input/sources/tev-000066.yaml

```diff
@@ -4,7 +4,8 @@ common_name: HESS J1420-607
 gamma_names: [HESS J1420-607]
 fermi_names: []
 other_names:
-- PSR J1420-6048
```

Comment:
In HESS source catalog HESS J1420-607 corresponds to Kookaburra -> See comment on source 65 as well

Reply:
Thanks for spotting this. Yes, HESS J1420 should also have "Kookaburra" in `other_names`